### DISCUSSION TITLE
Weights & Biases integration with Pycaret

### DISCUSSION THREAD

**Discussion by victor-mariano-leite** [pycaret/pycaret#1241] [Q&A]
Has someone tried to integrate W&B experiment versioning, trackings with Pycaret?

**Comment by pycaret** [comment]
@victor-mariano-leite We have integration with `MLflow` which essentially does the same thing. 

Checkout: http://pycaret.org/mlflow/

**Reply by victor-mariano-leite** [reply]
Actually I've experimented a bit :) works well and seems quite good, but MLFlow still lacks some features like hardware usage trackings, artifacts and data versioning git-like, i've been experimenting W&B recently because encapsulates lot's of these features in a concise way, with centralized dashboards (with very good default visualizations) reports etc. Maybe it would be interesting to add this flexibility to Pycaret

**Comment by RumiAllbert** [comment]
I saw that in pycaret 3.0 there is "New experiment logging API with MLFLow, Weights & Biases, DagsHub and CometML support". Will documentation for this be published?
